packages/backend/server/routes/stats.get.ts

```diff
@@ -1,40 +1,23 @@
-import { sha256 } from "ohash"
+import { sha256 } from "ohash";
 
 export default eventHandler(async (event) => {
   try {
     const binding = useBinding(event);
 
-    let started = false
-
-    let cursor: string | undefined;
+    const query = getQuery(event);
 
+    let cursor = query.cursor || undefined;

```

Comment:
how can we use the next cursor? i tried but apparently it does not work!

Reply:
It does work